.eslintrc

```diff
@@ -51,6 +51,14 @@
         "properties": "never"
       }
     ],
+    "capitalized-comments": [
+      "error",
+      "always",
+      {
+        "ignorePattern": "disable-bad-pattern-check|fall-through",
```

Comment:
Can you explain the use of these patterns? Can you show where we have used it in the codebase?

Reply:
**fall-through** is used here(https://github.com/oppia/oppia/blob/b3be90d330db7974fdb90e45b33ddd7fcfa1d189/extensions/interactions/FractionInput/directives/fraction-input-validation.service.ts#L175)
**disable-bad-pattern-check** is used to disable custom lint checks. So it used at many places.

Reply:
Thanks for the info, @Hudda!